интересно эту хуйню ещё можно смержить или уже всё

AshtonFox opened the pull request.

FighterX2500 commented:
бля ну ты ебанутый конечно, я видел кучу сообщений поступающих на почту про твой пулл реквест и новые коммиты, но что бы билдос содержать это мем нахуй.

FighterX2500 commented:
>We can't automatically merge this pull request.

Надо уде вручную.

Novacat committed:
Fixes Pallette with Rosettes

Rosettes used a bit lighter shade of grey, which did not work well with the way markings are applied. This fixes that.

Novacat committed:
Merge pull request #5000 from Novacat/nova-survival

Fixes Pallette with Rosettes

Novacat committed:
SMESs no longer turn off when out of power

Ported from Baystation PR #11973, Mirrored to Polaris PR

Novacat committed:
Merge pull request #5002 from Novacat/nova-survival

SMESs no longer turn off when out of power

Novacat committed:
Merge pull request #4984 from Heroman3003/jello-equivalent-of-glass-bones

Undoes some VOREStation promethean nerfs

Novacat committed:
Merge pull request #4881 from Heroman3003/borg-resization

Borgs are now in on fun of size-related features

Novacat committed:
Merge pull request #4996 from Heroman3003/safety-first

Fixes xenoarch spawning on station levels

Heroman3003 committed:
Fixes NSFW batteries not being rechargable

Novacat committed:
Merge pull request #5005 from Heroman3003/not-safe-for-chargers

Fixes NSFW batteries not being rechargable

Heroman3003 committed:
Unruntimes chems

Novacat committed:
Merge pull request #5006 from Heroman3003/unruntiming

Unruntimes chems

Heroman3003 committed:
Allows cyborgs and AIs to appear in character directory

Novacat committed:
Merge pull request #5007 from Heroman3003/directoring-the-borgfriends

Allows cyborgs and AIs to appear in character directory

Heroman3003 committed:
Adds AI shell landmark to the map

Novacat committed:
Merge pull request #5008 from Heroman3003/ai-shell-landmark-for-future

Adds AI shell landmark to the map

Heroman3003 committed:
Fixes ghost shuttles causing runtimes

Heroman3003 committed:
Fixes micros not getting abducted properly

Novacat committed:
Merge pull request #5009 from Heroman3003/shuttle-unruntiming

Fixes ghost shuttles causing runtimes

Novacat committed:
Merge pull request #5010 from Heroman3003/abductineringfgjfhlkljfghnfdsfaRGGFDGH

Fixes micros in pockets not getting abducted properly

Heroman3003 committed:
Makes nanoui apply properly to roundstart clients

Novacat committed:
Merge pull request #5011 from Heroman3003/rrhgfdxn

Makes nanoui apply properly to roundstart clients

Heroman3003 committed:
Changes shakers to use initialize

Novacat committed:
Merge pull request #5012 from Heroman3003/shakers-must-use-initialize-INITIALIZE

Changes shakers to use initialize

Heroman3003 committed:
Fixes organizations runtiming

Heroman3003 committed:
Fixes runtime in announcer AI

SpadesNeil committed:
Merge pull request #4957 from Heroman3003/lower-respawn-timer

Lowers respawn timer back to 1 minute

SpadesNeil committed:
Merge pull request #4897 from Heroman3003/bone-zone

Added a remains option for digesty vore

Novacat committed:
Fixes #5016

Polaris code changes ended up being not compatable with vorestation stuff, this should correct that.

Novacat committed:
Merge pull request #5014 from Heroman3003/fix-for-org-runtimes

Fixes organizations runtiming

Novacat committed:
Merge pull request #5015 from Heroman3003/overtime-overtime-overtime-over

Fixes runtime in announcer AI

Novacat committed:
Merge pull request #5017 from Novacat/nova-survival

Fixes #5016

Heroman3003 committed:
Fixes missing shaker sprite

Novacat committed:
Merge pull request #5019 from Heroman3003/shaker-fix

Fixes missing shaker sprite

Heroman3003 committed:
Fixes step code, again

Heroman3003 committed:
Allows help-intent step with buckled mobs

Novacat committed:
Fixes Evian's Egg Implant

It was in a different part of the file so I missed it.

Heroman3003 committed:
Forgot vorestation edits

Novacat committed:
Merge pull request #5023 from Novacat/nova-survival

Fixes Evian's Egg Implant

Novacat committed:
Merge pull request #5021 from Heroman3003/steppy-steppy-on-the-meow

Fixes step code, again

Heroman3003 committed:
Fixes notify transcore checking for presense of backup

Heroman3003 committed:
Removes unnecesary var

Novacat committed:
Merge pull request #5024 from Heroman3003/proper-transcoring

Fixes notify transcore checking for presense of backup

Zap-zapper committed:
Size control implant fix

Novacat committed:
Merge pull request #5026 from Zap-zapper/master

Size control implant fix

Neerti committed:
Fixes closet initialization

Heroman3003 committed:
Makes solars and substation smes start with output off

Heroman3003 committed:
Atmos sub starts on

Neerti committed:
Arrivals Power Fix

Neerti committed:
Adds cancel button to kick button

Mewchild committed:
Adds a colorable cloak in loadout (#6040)

VirgoBot committed:
Adds a colorable cloak in loadout

Neerti committed:
Fixes weather sound z-level desyncs from shuttles

Novacat committed:
Merge pull request #5030 from VOREStation/upstream-merge-6064

[MIRROR] Arrivals Power Fix

Novacat committed:
Merge pull request #5031 from VOREStation/upstream-merge-6034

[MIRROR] Adds cancel button to kick button

Novacat committed:
Merge pull request #5028 from VOREStation/upstream-merge-6033

[MIRROR] Fixes closet initialization

Novacat committed:
Merge pull request #5032 from VOREStation/upstream-merge-6040

[MIRROR] Adds a colorable cloak in loadout

Novacat committed:
Merge pull request #5033 from VOREStation/upstream-merge-6035

[MIRROR] Fixes weather sound z-level desyncs from shuttles

Heroman3003 committed:
Some cyborg adjustments for map

Neerti committed:
Base features for Precursotech and Anomalotech Expansion

Novacat committed:
Update materials.dm

Novacat committed:
Merge pull request #5035 from VOREStation/upstream-merge-6023

[MIRROR] Base features for Precursotech and Anomalotech Expansion

Neerti committed:
Merge pull request #6024 from Mechoid/Precursotech_Sleepers

Precurso / Anomalotech Sleepers

VirgoBot committed:
Precurso / Anomalotech Sleepers

Neerti committed:
Merge pull request #6027 from Mechoid/Precursotech_Sword

Anomalotech Charge Sword

VirgoBot committed:
Anomalotech Charge Sword

Neerti committed:
Merge pull request #6028 from Mechoid/Precursotech_Tools

Precursotech Tools

VirgoBot committed:
Precursotech Tools

Neerti committed:
Fixes tesla logic bug

Novacat committed:
Merge pull request #5036 from VOREStation/upstream-merge-6024

[MIRROR] Precurso / Anomalotech Sleepers

Novacat committed:
Merge pull request #5037 from VOREStation/upstream-merge-6027

[MIRROR] Anomalotech Charge Sword

Neerti committed:
Changes shakers to use initialize

Neerti committed:
Fixes runtimes from organizations catalogue

Neerti committed:
Fixed announcer AI powersupply runtime

Neerti committed:
Merge pull request #6073 from Heroman3003/conveyor-boogalo

Fixes newly constructed conveyors not working

Neerti committed:
Merge pull request #6074 from Heroman3003/fix-the-misalign

Fixes misaligned circuitry gloves sprite

Neerti committed:
Merge pull request #6076 from Heroman3003/double-shock-fix

Fixed the vendor machines zapping twice

VirgoBot committed:
Fixed the vendor machines zapping twice

Neerti committed:
Merge pull request #6077 from Heroman3003/meson-runtime-fix

Fixes glasses with overlays runtiming on clientless mobs

VirgoBot committed:
Fixes misaligned circuitry gloves sprite

VirgoBot committed:
Fixes newly constructed conveyors not working

VirgoBot committed:
Fixes glasses with overlays runtiming on clientless mobs

Neerti committed:
Merge pull request #6037 from Heroman3003/report-the-crate-cost-lazy-central-interns

Fixes export reports not mentioning crates' cost

Neerti committed:
Allows restriction of z-levels from spawning xenoarch finds

VirgoBot committed:
Fixes export reports not mentioning crates' cost

Novacat committed:
Merge pull request #5038 from VOREStation/upstream-merge-6028

[MIRROR] Precursotech Tools

Neerti committed:
Merge pull request #6066 from Heroman3003/request-relabeling

Allows the request console ID to actually be changed

Neerti committed:
SMESs no longer turn off when out of power

Neerti committed:
Merge pull request #6078 from Heroman3003/unbluespacing-vehicles

Fixes resisting out of vehicles causing problems

VirgoBot committed:
Fixes resisting out of vehicles causing problems

VirgoBot committed:
Allows the request console ID to actually be changed

Novacat committed:
Merge pull request #5039 from VOREStation/upstream-merge-6063

[MIRROR] Fixes tesla logic bug

Novacat committed:
Merge pull request #5040 from VOREStation/upstream-merge-6069

[MIRROR] Changes shakers to use initialize

Novacat committed:
Merge pull request #5041 from VOREStation/upstream-merge-6076

[MIRROR] Fixed the vendor machines zapping twice

Novacat committed:
Merge pull request #5042 from VOREStation/upstream-merge-6074

[MIRROR] Fixes misaligned circuitry gloves sprite

Novacat committed:
Merge pull request #5043 from VOREStation/upstream-merge-6073

[MIRROR] Fixes newly constructed conveyors not working

Novacat committed:
Merge pull request #5044 from VOREStation/upstream-merge-6071

[MIRROR] Fixed announcer AI powersupply runtime

Neerti committed:
Merge pull request #6080 from Mechoid/FixAccidentalAlchemy

Fixes My Idiocy

Novacat committed:
Merge pull request #5045 from VOREStation/upstream-merge-6070

[MIRROR] Fixes runtimes from organizations catalogue

VirgoBot committed:
Fixes My Idiocy

Novacat committed:
Update xenoarch.dm

Novacat committed:
Update maps.dm

Novacat committed:
Merge pull request #5046 from VOREStation/upstream-merge-6077

[MIRROR] Fixes glasses with overlays runtiming on clientless mobs

Novacat committed:
Merge pull request #5047 from VOREStation/upstream-merge-6037

[MIRROR] Fixes export reports not mentioning crates' cost

Novacat committed:
tired lynx mrowl

Novacat committed:
Merge pull request #5049 from VOREStation/upstream-merge-6078

[MIRROR] Fixes resisting out of vehicles causing problems

Novacat committed:
Merge pull request #5050 from VOREStation/upstream-merge-6067

[MIRROR] SMESs no longer turn off when out of power

Novacat committed:
Merge pull request #5051 from VOREStation/upstream-merge-6066

[MIRROR] Allows the request console ID to actually be changed

Novacat committed:
Merge pull request #5052 from VOREStation/upstream-merge-6080

[MIRROR] Fixes My Idiocy

Novacat committed:
Merge pull request #5048 from VOREStation/upstream-merge-6065

[MIRROR] Allows restriction of z-levels from spawning xenoarch finds

Heroman3003 committed:
Fixes moving items downstairs, virgo edition

Novacat committed:
Merge pull request #5053 from Heroman3003/downstairs-pull

Fixes pulling items downstairs, virgo edition

Heroman3003 committed:
Fixes mob pathing for wilderness and V2

Neerti committed:
Merge pull request #6082 from Heroman3003/sleeper-a-roo

Fixes several issues with sleepers

Neerti committed:
Merge pull request #6083 from Heroman3003/runtime-lead-sand-whatever

Fixes runtiming with mineral turfs

VirgoBot committed:
Fixes several issues with sleepers

VirgoBot committed:
Fixes runtiming with mineral turfs

Novacat committed:
Merge pull request #5055 from VOREStation/upstream-merge-6082

[MIRROR] Fixes several issues with sleepers

Novacat committed:
Merge pull request #5056 from VOREStation/upstream-merge-6083

[MIRROR] Fixes runtiming with mineral turfs

Novacat committed:
Merge pull request #5054 from Heroman3003/mob-pathfix

Fixes mob pathing for wilderness and V2

Heroman3003 committed:
Removes Command from appearing for players who had it removed

Sebbe9123 committed:
Fixes pathing on a lot of maps, removes snek from beach.

Heroman3003 committed:
Replaces regulators in atmos with pumps where needed

Novacat committed:
Merge pull request #5058 from Heroman3003/defunct-pto

Removes Command PTO from appearing for players who had it

Novacat committed:
Merge pull request #5059 from Heroman3003/back-to-the-pumps

Replaces regulators in atmos with pumps

Neerti committed:
Fixes some signaler bugs

Neerti committed:
Merge pull request #6085 from Heroman3003/apc-cost-fix

Fixes apc frame giving more steel than needed

VirgoBot committed:
Fixes apc frame giving more steel than needed

Neerti committed:
Merge pull request #6086 from Heroman3003/welder-fires

Fixes self-refueling welders starting fires when off

VirgoBot committed:
Fixes self-refueling welders starting fires when off

Neerti committed:
Merge pull request #6087 from Heroman3003/pipe-buckle-general

Fixes some bugs related to buckling

VirgoBot committed:
Fixes some bugs related to buckling

Novacat committed:
Conflict Resolution

Novacat committed:
Merge pull request #5061 from VOREStation/upstream-merge-6085

[MIRROR] Fixes apc frame giving more steel than needed

Novacat committed:
Merge pull request #5063 from VOREStation/upstream-merge-6086

[MIRROR] Fixes self-refueling welders starting fires when off

Novacat committed:
Merge pull request #5064 from VOREStation/upstream-merge-6087

[MIRROR] Fixes some bugs related to buckling

Neerti committed:
Merge pull request #6088 from Heroman3003/infinite-cash-no

Fixes money duping bug

VirgoBot committed:
Fixes money duping bug

Novacat committed:
Merge pull request #5062 from VOREStation/upstream-merge-6089

[MIRROR] Fixes some signaler bugs

Novacat committed:
Merge pull request #5065 from VOREStation/upstream-merge-6088

[MIRROR] Fixes money duping bug

Heroman3003 committed:
Fixes moth antennae+wing combo not being colorable

Novacat committed:
Merge pull request #5067 from Heroman3003/mothing-around

Fixes moth antennae+wing combo not being colorable

Heroman3003 committed:
Fixes snakess being aggressive

Novacat committed:
Merge pull request #5068 from Heroman3003/cool-the-snek

Fixes snakes being aggressive

Sebbe9123 committed:
Hopefully fixes every path

plz game, I didn't have fun ;w;

Sebbe9123 committed:
Game b weird

Sebbe9123 committed:
Merge remote-tracking branch 'Upstream/master'

Heroman3003 committed:
Replaces drone back with mech

Sebbe9123 committed:
Merge branch 'master' into mobpathfix

LiquidFirefly committed:
Add files via upload

Heroman3003 committed:
Removes trash piles hidden inside wall pockets

Novacat committed:
Merge pull request #5072 from LiquidFirefly/WHY-THE-FUCK-THIS-LINE-NOT-IN,-HOW-DID-NOBODY-NOTICE-JESUS

Fixes spiderleg coloring.

Heroman3003 committed:
Fixes harpoon targeting obstructed tiles

Novacat committed:
Merge pull request #5075 from Heroman3003/harpoon-fixing

Fixes bluespace harpoon targeting obstructed tiles

Neerti committed:
Merge pull request #6091 from Heroman3003/yes-ye-yess

Fixes *yes

VirgoBot committed:
Fixes *yes

Novacat committed:
Merge pull request #5070 from Heroman3003/advanced-dark-gygax-attacks

Replaces drone back with mech

Novacat committed:
Merge pull request #5029 from Heroman3003/solar-output-adjust

Makes solars and substation smes start with output off

Novacat committed:
Merge pull request #5034 from Heroman3003/landmarkening

Some cyborg adjustments for map

Novacat committed:
Merge pull request #5076 from VOREStation/upstream-merge-6091

[MIRROR] Fixes *yes

Heroman3003 committed:
Merge branch 'master' into untrashing-stashes

Heroman3003 committed:
Merge branch 'master' into untrashing-stashes

Heroman3003 committed:
Fixes merge conflicts

Heroman3003 committed:
Fixes windows in mapping tool not appearing fulltile

Heroman3003 committed:
Fixes cryopod not allowing people to change their mind

Neerti committed:
Fixes fireaxe cabinets

Neerti committed:
Merge pull request #6093 from Heroman3003/zpipe-fix

Fixes newly constructed vertical pipes being invisible

VirgoBot committed:
Fixes newly constructed vertical pipes being invisible

Novacat committed:
Merge pull request #5078 from Heroman3003/sleepless-in-adephagia

Fixes cryopod not allowing people to change their mind

Novacat committed:
Merge pull request #5077 from Heroman3003/fulltile-map-windows-virgo-style

Preparation for the fix for windows in mapping tool not appearing fulltile

Novacat committed:
Merge pull request #5079 from VOREStation/upstream-merge-6093

[MIRROR] Fixes newly constructed vertical pipes being invisible

Novacat committed:
Conflict Resolution

Novacat committed:
Merge pull request #5080 from VOREStation/upstream-merge-6092

[MIRROR] Fixes fireaxe cabinets

Neerti committed:
Merge pull request #6094 from Heroman3003/anti-magic-markers

Fixes markers and crayons drawing from a distance

VirgoBot committed:
Fixes markers and crayons drawing from a distance

Novacat committed:
Merge pull request #5081 from VOREStation/upstream-merge-6094

[MIRROR] Fixes markers and crayons drawing from a distance

Neerti committed:
Merge pull request #6095 from Heroman3003/unduping-synths

Fixes synthesisers inserting more materials than they had

VirgoBot committed:
Fixes synthesisers inserting more materials than they had

Novacat committed:
Merge pull request #5082 from VOREStation/upstream-merge-6095

[MIRROR] Fixes synthesisers inserting more materials than they had

Sebbe9123 committed:
Did the doo as requested

Sebbe9123 committed:
Forgot I need to do the dme

Sebbe9123 committed:
aaaand the dme didn't work first time???

LiquidFirefly committed:
Research QoL changes (#6042)

* Research QoL changes

* Reverts changes relating to blanket print times and Protolathe storage

* Actually changes the design print time.

Why do you hate me github.

* Fixes an enter key error caused by the editor.

* Actually does all of the things I needed to do like a smart person

Which apparently I am not.

* Fixes a newline

Because goddamn it the online editor hates me too.

VirgoBot committed:
Research QoL changes

Novacat committed:
Merge pull request #5083 from VOREStation/upstream-merge-6042

[MIRROR] Research QoL changes

Novacat committed:
Merge pull request #5073 from Heroman3003/untrashing-stashes

Removes trash piles hidden inside wall pockets

Heroman3003 committed:
Fixes some intercoms

Heroman3003 committed:
Fixes more intercoms

Novacat committed:
Merge pull request #5084 from Heroman3003/fix-the-intercoms-already

Some intercom QoL

Atermonera committed:
Precursotech Janus

Mechoid committed:
Adds Robobags (#6059)

* Adds Robobags

* Finishes robobags

* FixFix

VirgoBot committed:
Adds Robobags

Novacat committed:
Merge pull request #5086 from VOREStation/upstream-merge-6059

[MIRROR] Adds Robobags

Heroman3003 committed:
Fixes wheelchairs. For real.

Novacat committed:
Conflict Resolution

Novacat committed:
Merge pull request #5087 from Heroman3003/fix-the-wheelchairs-for-real

Fixes wheelchairs. For real.

Novacat committed:
Merge pull request #5085 from VOREStation/upstream-merge-6026

[MIRROR] Precursotech Janus

Atermonera committed:
Starts work on Organs affecting Reagent Processing

Atermonera committed:
Merge pull request #6100 from Heroman3003/silicate-fix

Fixes silicate making fulltile windows invisible

VirgoBot committed:
Fixes silicate making fulltile windows invisible

Atermonera committed:
Merge pull request #6103 from Heroman3003/convinienct-suiting

Allows voidsuits to be modified in hands

VirgoBot committed:
Allows voidsuits to be modified in hands

Novacat committed:
Update modifiers_misc.dm

Atermonera committed:
Merge pull request #6106 from chaoko99/Pullbox

ore bags will now automatically pick up ore and dump them into a box if one is being pulled.

Atermonera committed:
Merge pull request #6101 from Heroman3003/icon-window-fix

Fixes mapping tool icons for fulltile windows

Novacat committed:
Merge pull request #5088 from VOREStation/upstream-merge-6100

[MIRROR] Fixes silicate making fulltile windows invisible

VirgoBot committed:
Fixes mapping tool icons for fulltile windows

VirgoBot committed:
ore bags will now automatically pick up ore and dump them into a box if one is being pulled.

Atermonera committed:
Merge pull request #6097 from Heroman3003/mice-for-goo

Fixes promethean mice lunchboxes

VirgoBot committed:
Fixes promethean mice lunchboxes

Atermonera committed:
Merge pull request #6090 from Mechoid/Hopefully_Fixes_Ghost_Checks

Hopefully fixes ghost presence checks.

VirgoBot committed:
Hopefully fixes ghost presence checks.

Atermonera committed:
Merge pull request #6098 from Heroman3003/laserpoint-fix

Fixes laser pointers targeting through walls

VirgoBot committed:
Fixes laser pointers targeting through walls

Novacat committed:
Merge pull request #5090 from VOREStation/upstream-merge-6103

[MIRROR] Allows voidsuits to be modified in hands

Atermonera committed:
Changes way syringes interact with containers

Atermonera committed:
Merge pull request #6099 from Heroman3003/fcookie-runtime-fix

Fixes fortune cookie recipe runtime

VirgoBot committed:
Fixes fortune cookie recipe runtime

Novacat committed:
Merge pull request #5089 from VOREStation/upstream-merge-6081

[MIRROR] Starts work on Organs affecting Reagent Processing

Novacat committed:
Merge pull request #5092 from VOREStation/upstream-merge-6106

[MIRROR] ore bags will now automatically pick up ore and dump them into a box if one is being pulled.

Novacat committed:
Merge pull request #5091 from VOREStation/upstream-merge-6101

[MIRROR] Fixes mapping tool icons for fulltile windows

Novacat committed:
Merge pull request #5093 from VOREStation/upstream-merge-6097

[MIRROR] Fixes promethean mice lunchboxes

Novacat committed:
Conflict Resolution

Novacat committed:
Merge pull request #5094 from VOREStation/upstream-merge-6090

[MIRROR] Hopefully fixes ghost presence checks.

Novacat committed:
Merge pull request #5095 from VOREStation/upstream-merge-6098

[MIRROR] Fixes laser pointers targeting through walls

Atermonera committed:
Makes transfer always occur if nobody is playing.

Novacat committed:
Conflict Resolution

Novacat committed:
Merge pull request #5097 from VOREStation/upstream-merge-6099

[MIRROR] Fixes fortune cookie recipe runtime

Novacat committed:
Update syringes.dm

Novacat committed:
Update syringes.dm

Novacat committed:
Merge pull request #5098 from VOREStation/upstream-merge-6109

[MIRROR] Makes transfer always occur if nobody is playing.

Novacat committed:
Merge pull request #5096 from VOREStation/upstream-merge-6096

[MIRROR] Changes way syringes interact with containers

Novacat committed:
Vulpkanin Voidsuit Sprites

- Explorer and Pilot Voidsuits
- Ported from CitadelRP

Novacat committed:
Merge pull request #5099 from Novacat/nova-survival

Vulpkanin Voidsuit Sprites

Sebbe9123 committed:
unedits thing as requested

Novacat committed:
Update beach.dmm

Novacat committed:
Merge pull request #5069 from Sebbe9123/mobpathfix

Hopefully fixes all paths here and there

Heroman3003 committed:
Fixes outpost intercoms, removes redundant maps

Heroman3003 committed:
Fixes more intercom directions on station itself

Novacat committed:
Merge pull request #5102 from Heroman3003/outpost-intercoms

Fixes more intercoms, removes redundant maps

Neerti committed:
Merge pull request #6119 from Heroman3003/sticky-substance

Fixes nurse spiders getting stuck in infinite loop

VirgoBot committed:
Fixes nurse spiders getting stuck in infinite loop

Novacat committed:
Merge pull request #5103 from VOREStation/upstream-merge-6119

[MIRROR] Fixes nurse spiders getting stuck in infinite loop

Neerti committed:
Merge pull request #6112 from Heroman3003/unredundanting

Removes redundant fix for a bug that never existed

VirgoBot committed:
Removes redundant fix for a bug that never existed

Novacat committed:
Merge pull request #5105 from VOREStation/upstream-merge-6112

[MIRROR] Removes redundant fix for a bug that never existed

Atermonera committed:
Merge pull request #6102 from Heroman3003/taping-it-all-up

Fixes taperolls not being usable on tiles with directional windows

VirgoBot committed:
Fixes taperolls not being usable on tiles with directional windows

Novacat committed:
Merge pull request #5106 from VOREStation/upstream-merge-6102

[MIRROR] Fixes taperolls not being usable on tiles with directional windows

Atermonera committed:
Changelog, because I'm not too busy to do this now!

Novacat committed:
Merge pull request #5107 from VOREStation/upstream-merge-6111

[MIRROR] Changelog, because I'm not too busy to do this now!
